internal/layer2/announcer.go

```diff
@@ -174,6 +176,13 @@ func (a *Announce) SetBalancer(name string, ip net.IP) {
 		return
 	}
 
+	a.ipRefcnt[ip.String()]++
+	if a.ipRefcnt[ip.String()] > 1 {
+		// Multiple services are using this IP, so there's nothing
+		// else to do right now.
```

Comment:
I'm super-unfamiliar with this codebase, but have just started using metallb more extensively and have run into #335.

Could this be caused due to this `if` brace?